python/ray/_private/services.py

```diff
@@ -1879,17 +1879,16 @@ def build_java_worker_command(
     pairs.append(("ray.home", RAY_HOME))
     pairs.append(("ray.logging.dir", os.path.join(session_dir, "logs")))
     pairs.append(("ray.session-dir", session_dir))
+    # Add ray jars path to java classpath
+    ray_jars = os.path.join(get_ray_jars_dir(), "*")
```

Comment:
maybe "*.jar"?

Reply:
This line is moved to here. And according to the last comment, we should support not only jar.